### PR TITLE
[Pushover] Updates to reflect language used in the API

### PR DESCRIPTION
**Pull request by kolcun**
- use ApiKey instead of ApiToken to match the Pushover API
- Reordering of the action section - show current actions first, move depreciated actions later
- Fixing the deprecated syntax highlighting
- Added default device to the table
- Removed the note about Builder pattern, not sure that it's really needed

### CODE REVIEW

File: bundles/action/org.openhab.action.pushover/README.md [hunk 1]
```diff
@@ -5,49 +5,33 @@ The Pushover action service allows you to notify mobile devices of a message usi
...
-- `Deprecated ~~pushover(String apiToken, String userKey, String message, String device, String title, String url, String urlTitle, int priority, String soundFile)~~`
-- `Deprecated ~~pushover(String apiToken, String userKey, String message, String device, String title, String url, String urlTitle, int priority, String soundFile, String attachment)~~`
 - `sendPushoverMessage(Pushover pushover)`
 
-The action calls have to be configured in the above sequence. The builder pattern is used to provide the parameters. The most parameters are optional and may have a default value. See the examples how to call use the pushover action.
-In this case any default values from `services/pushover.cfg` will be used.
+Most parameters are optional and may have a default value. In this case any default values from `services/pushover.cfg` will be used.
 Note that you cannot use a null value for int priority.
-
+See the examples section for detials on how use the pushover action.
```
Comment (9037568) [inline bundles/action/org.openhab.action.pushover/README.md]: "details"

File: bundles/action/org.openhab.action.pushover/README.md [hunk 2]
```diff
@@ -5,49 +5,33 @@ The Pushover action service allows you to notify mobile devices of a message usi
...
-| defaultExpire   | 3600    | No                                                | -                                   | When priority is 2 (high priority), how long (in seconds) to continue resending messages until acknowledged            |
-| attachment      |         | No                                                | withAttachment(String attachment)   | The full path of a JPEG image attachment to be pushed with the message.                                                |
-| contentType     |         | No                                                | withContentType(String contentType) | Content type, ie  "image/png"                                                                                          |
+None of the configuration parameters are required as you can specify required configuration items in the action call, but you must at least provide an *API Key*, *User/Group Key* and a *Message* in some manner before a notification can be pushed.
```
[the remark this answers is not in the document]
Comment (kolcun) [reply bundles/action/org.openhab.action.pushover/README.md]: Agree, I'll adjust. The api seems to use a mix of the terms interchangeably.

File: bundles/action/org.openhab.action.pushover/README.md [hunk 3]
```diff
@@ -5,49 +5,33 @@ The Pushover action service allows you to notify mobile devices of a message usi
...
+
+| Property        | Default | Required                                                      | method to set                       | Description                                                                                                            |
+|-----------------|---------|:-------------------------------------------------------------:|-------------------------------------|------------------------------------------------------------------------------------------------------------------------|
+| defaultTimeout  | 10000   | No                                                            | -                                   | Timeout in milliseconds for the connection to pushover.net                                                             |
+| defaultApiKey   |         | Yes, if using action call without calling withApiKey() method | withApiKey(String apiKey)           | Pushover [API key](https://pushover.net/api) to send to devices                                                        |
```
[the remark this answers is not in the document]
Comment (kolcun) [reply bundles/action/org.openhab.action.pushover/README.md]: Use Token, not Key.

File: bundles/action/org.openhab.action.pushover/README.md [hunk 4]
```diff
@@ -5,49 +5,33 @@ The Pushover action service allows you to notify mobile devices of a message usi
...
+| defaultUser     |         | Yes, if using action call without calling withUser() method   | withUser(String user)               | Pushover User or Group key (not e-mail address) of your user (or you) to send to devices.                              |
+| defaultDevice   |         | No                                                            | withDevice(String device)           | Pushever device which will receive the notification                                                                    |               |
```
[the remark this answers is not in the document]
Comment (kolcun) [reply bundles/action/org.openhab.action.pushover/README.md]: removed

**Comment by kolcun** [comment]
Updated changes based on comments - thanks.

**Comment by 9037568** [comment]
Thanks, @kolcun !

**Pull request merged**